2-mandatory/5-reading-list.js

```diff
@@ -47,6 +73,21 @@ As an example for this exercise, you might do the following steps
 - Modify the books so that they have the correct alreadyRead value
 - All tests should turn green!!
 **/
+function readBooks()
+{
+    books.forEach(book => 
+    {
+        if(book.alreadyRead)
+        {
+            console.log("You've already read " + book.title + " by " + book.author + ".")
```

Comment:
Great job! you could also use backtick ` quote and using ${ }  for the variables.